SI/Sem.02/Pract.02/task07.cpp

```diff
@@ -0,0 +1,31 @@
+// ConsoleApplication7.cpp : This file contains the 'main' function. Program execution begins and ends there.
+//
+
+#include <iostream>
+using namespace std;
+
+int main()
+//�� �� ������ ��������, ����� ���� �� ��������� ������ �, ��� � ����� �����, ������� ����������� ������, ��� � ������ �����,
+//������� ����������� �����, � ��� �� � �����, �� ������ ��������� �� ��������� ������.
+{
+	char symbol;
+	cout << "Enter a symbol: ";
+	cin >> symbol;
+	if ('a' <= symbol <= 'z')
```

Comment:
Re-visit this task.
- 'a' <= symbol <= 'z' does this work?
- if you have 2 conditions -> logical operator &&
- Check operator += and -=